src/common/FrameManager.ts

```diff
@@ -261,9 +265,10 @@ export class FrameManager extends EventEmitter {
     }
 
     const frame = this._frames.get(event.targetInfo.targetId);
-    const session = Connection.fromSession(this._client).session(
-      event.sessionId
-    );
+    const connection = Connection.fromSession(this._client);
+    if (!connection) return;
```

Comment:
should we throw an error here instead? It looks like we normally expect the connection and session below to be there (unless smth goes very wrong).